src/components/logo/index.js

```diff
@@ -7,10 +7,9 @@ const Logo = props => {
       viewBox="0 0 1141 623"
       className={props.className}
     >
-      <path
-        d="M82.5 299.7H24.4L.2 3.6h52.1l6 193.4L66.6 3.6h47l8.8 194.6 6-194.6H177l-24.3 296.1h-57L89.3 131l-6.8 168.7zM300.9 59.3v185.9c0 18.3-4.7 32.4-14.1 42.4-9.4 9.9-22.5 14.9-39.2 14.9h-8.4c-17.5 0-30.8-5-40-14.9s-13.7-24.1-13.7-42.4V59.3c0-39 17.9-58.5 53.7-58.5h8c35.7 0 53.7 19.5 53.7 58.5zm-51.8 190.2V53.7c0-5.8-2-8.8-6-8.8-3.7 0-5.6 2.9-5.6 8.8v195.8c0 5.3 1.9 8 5.6 8 4 0 6-2.7 6-8zm66.7 50.2V3.6h57.7l8.8 143.7L391 3.6h56.1v296.1H405V134.1l-14.7 165.6h-17.9l-15.1-165.6v165.6h-41.5zM462.1 3.6h85.2V49h-35.8v72h30.2v47h-30.2v85.6h37v46.2h-86.4V3.6zm100.5 0h50.9L633 164l-.4-160.4h41.8v296.1h-46.6l-23.5-177.5.4 177.5h-42.2V3.6zM82.5 620H24.4L.2 323.9h52.1l6 193.4 8.4-193.4h47l8.8 194.6 6-194.6h48.6L152.6 620H95.7l-6.4-168.7L82.5 620zm104.9 0V323.9H238v115h10.7v-115h50.1V620h-50.1V491.1H238V620h-50.6zm241.8-240.4v185.9c0 18.3-4.7 32.4-14.1 42.4-9.4 9.9-22.5 14.9-39.2 14.9h-8.4c-17.5 0-30.8-5-40-14.9s-13.7-24.1-13.7-42.4V379.6c0-39 17.9-58.5 53.7-58.5h8c35.8 0 53.7 19.5 53.7 58.5zm-51.7 190.3V374.1c0-5.8-2-8.8-6-8.8-3.7 0-5.6 2.9-5.6 8.8v195.8c0 5.3 1.9 8 5.6 8 4-.1 6-2.7 6-8zM482.2 620V323.9h72c26 0 39 14.2 39 42.6V575c0 15.1-3.4 26.4-10.3 33.8-6.9 7.5-16.2 11.2-27.9 11.2h-72.8zm49.4-255.5v213.7h4.8c4 0 6-3.3 6-9.9V373.7c0-6.1-2-9.2-6-9.2h-4.8zm76.6-40.6h85.2v45.4h-35.8v72h30.2v47h-30.2v85.6h37V620h-86.4V323.9zm160.2 298.9h-11.5c-32.9 0-49.4-15.3-49.4-45.8v-77.6h49.4v74c0 5.3 1.7 8 5.2 8 3.7 0 5.6-2.8 5.6-8.4v-57.7c0-5-.5-8.9-1.6-11.5-1.1-2.7-3.4-5.6-7.2-8.8L733 472.8c-16.2-13.5-24.3-31.2-24.3-52.9v-52.5c0-30.8 16.8-46.2 50.5-46.2h11.5c31.8 0 47.8 14.1 47.8 42.2v65.3H770v-62.9c0-4.2-1.7-6.4-5.2-6.4s-5.2 2.1-5.2 6.4v50.5c0 6.1 2.7 11.7 8 16.7l25.5 22.3c10.3 8.5 17.5 16.8 21.5 24.9 4 8.1 6 18.5 6 31.2v58.9c-.1 35-17.4 52.5-52.2 52.5zm116.8-298.9V620h-50.5V323.9h50.5zm55.6 298.9h-1.2c-12.7 0-22.5-4.6-29.3-13.9-6.8-9.3-10.1-21-10.1-35V372.5c0-15.1 4.2-27.5 12.7-37 8.5-9.6 22-14.3 40.6-14.3h6.8c18.6 0 31.8 4.4 39.8 13.3 8 8.9 11.9 20.2 11.9 34V435h-48.6v-64.5c0-5-2-7.6-6-7.6-3.7 0-5.6 2.5-5.6 7.6v202.2c0 6.1 2.4 9.2 7.2 9.2s7.2-3.1 7.2-9.2v-88.4h-10.3v-38.6h58.1V620h-33.8c-3.4-12.2-5.2-18.4-5.2-18.7-9 14.3-20.4 21.5-34.2 21.5zm88.2-298.9h50.9l19.5 160.4-.4-160.4h41.8V620h-46.6l-23.5-177.5.4 177.5H1029V323.9z"
-        fill={props.fill}
-      />
+      <text x="0" y="150" font-size="220" fill={props.fill} class={props.className}>BLACKS</text>
+      <text x="0" y="360" font-size="220" fill={props.fill} class={props.className}>WHO</text>
+      <text x="0" y="570" font-size="220" fill={props.fill} class={props.className}>ENGINEER</text>
```

Comment:
I am _inordinately_ pleased to have figured out how to do this 😂

Reply:
...but that said, this is just a placeholder for now; I tried to make it look bad to encourage us to get an improved logo at some point